### ISSUE TITLE
Rendering artefacts on ground tiles

### ISSUE THREAD

**Issue by aggsol** [MasterQ32/VersaTile#34]
when looking from top. See attached screenshot.

![broken ground tiles](https://user-images.githubusercontent.com/719279/28892503-8695e380-77ce-11e7-9996-62caaa3d3315.png)

This might be a Z-Order/Buffer problem?

```
$ glxinfo | grep 'version'
server glx version string: 1.4
client glx version string: 1.4
GLX version: 1.4
OpenGL core profile version string: 4.5.0 NVIDIA 375.66
OpenGL core profile shading language version string: 4.50 NVIDIA
OpenGL version string: 4.5.0 NVIDIA 375.66
OpenGL shading language version string: 4.50 NVIDIA
OpenGL ES profile version string: OpenGL ES 3.2 NVIDIA 375.66
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.20
    GL_EXT_shader_implicit_conversions, GL_EXT_shader_integer_mix, 
```

**Comment by ikskuh** [comment]
Yeah this is a known problem right now... Maybe i can fix it with a simple `glPolygonOffset` call or just render the ground plane a bit below (like 0.25 "pixels" or something)
I'll try to fix it

**Comment by aggsol** [comment]
How about a skybox with a green bottom?

**Comment by ikskuh** [comment]
That wouldn't do the job of giving a "reference ground" you can work on... Also, you can turn the ground into a blueish grid in the preferences

**Comment by ikskuh** [comment]
Fixed in a56d99a0a01442524e90a9342251031c57faebc4
